utils/compression.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from __future__ import with_statement
+
+import logging
+import os
+import subprocess
+from string import Template
+
+logger = logging.getLogger(__name__)
+
+
+class Compressor(object):
```

Comment:
Maybe rename it to `BZ2Compressor` or something similar to know what compressor is used, just from the name